ats/pkb/pkb.py

```diff
@@ -0,0 +1,57 @@
+from ats.ast import nodes
+
+
+def process_follows(query, statements):
+    import json
+
+    print(json.dumps(query, indent=2))
+
+    return []
+
+
+def create_follows_dictionary(tree: nodes.ProgramNode):
+    parent_childrens_list = []
+    follows_dictionary = {}
+
+    def create_parent_childrens_list(node: nodes.ASTNode):
+        childrens = []
+        for i in range(len(node.children)):
+            create_parent_childrens_list(node.children[i])
+            childrens.append(node.children[i])
```

Comment:
tak na prawde na L17 mogliscie dac `children = node.children`, wyszloby na to samo